priv/repo/migrations/20200908163344_conversation_labels.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Phail.Repo.Migrations.ConversationLabels do
+  use Ecto.Migration
+
+  def change do
+    execute "create extension if not exists citext;"
+
+    create table(:labels) do
+      add :name, :citext
+    end
+
+    create table(:conversation_labels, primary_key: false) do
+      add :conversation_id, references(:conversations, on_delete: :delete_all)
+      add :label_id, references(:labels)
+    end
+
+    create unique_index(:conversation_labels, [:conversation_id, :label_id])
```

Comment:
would it also be useful to create an index for `label_conversations`? to answer the question of "what conversations all have label `X`"?

iirc when we were experimenting on monday we had indexes going both ways.

Reply:
Probably, yes. I think I was somewhat talking nonsense with my both ways around indexes, by the way. I think it's better to have a single joint index and build an index on conversation_id too.

I was going to add indexes as required when I figure out what the label queries look like.